### PR TITLE
Update with JSOBestieTemplate for NLPModelsJuMP

### PR DESCRIPTION
**Pull request by tmigot**
Automated update using JSOBestieTemplate

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -459,6 +459,9 @@ function _nlp_model(model::MOI.ModelLike)
     if isnothing(nlp_model)
       nlp_model = MOI.Nonlinear.Model()
     end
+    #attr = MOI.ListOfSupportedNonlinearOperators()
+    #ops = MOI.get(model, attr)
```
Comment (blegat) [inline src/utils.jl:463]: What is this for ?

File: src/moi_nlp_model.jl [hunk 2]
```diff
@@ -12,6 +12,19 @@ mutable struct MathOptNLPModel <: AbstractNLPModel{Float64, Vector{Float64}}
   counters::Counters
 end
 
+MOI.supports(model::Optimizer, ::MOI.UserDefinedFunction) = true
+
+function MOI.set(model::Optimizer, attr::MOI.UserDefinedFunction, args)
+    _init_nlp_model(model)
```
Comment (blegat) [inline src/moi_nlp_model.jl:18]: Maybe this could be a separate PR

File: src/utils.jl (hunk 1 above)
Comment (tmigot) [reply src/utils.jl]: Sorry @blegat , it was an automated PR that didn't run in a clean environment :s.
